Sprint #19.adoc

```diff
@@ -37,6 +37,7 @@
 == Questions
 
 * Do we want to manage quota on upload ? Ask that with INBOX and UX team
+** Answer : We don't care right now
```

Comment:
Cassandra storage limit ? In fact, we can rely on expiration to cleanup that space a bit. The only argument to implement this quota would be to prevent DoS (mass upload could be a problem)